ui/selectors/selectors.js

```diff
@@ -954,6 +984,19 @@ export function getPetnamesEnabled(state) {
   return petnamesEnabled;
 }
 
+export function getIsTokenNetworkFilterEqualCurrentNetwork(state) {
+  const chainId = getCurrentChainId(state);
```

Comment:
Would my hook be useful here?

Reply:
I dnt think so; the point of `getIsTokenNetworkFilterEqualCurrentNetwork` was to check if the user selected "current network" or "all networks" in the network filter and based on that we would either show the aggregatedPercentages and amounts for the current network or for all networks;

So now i added the display of the percentage component behind a feature flag here [33bcfad](https://github.com/MetaMask/metamask-extension/pull/28456/commits/33bcfad1337031c29bf8f80fd4ba9d87e15c3a68)

If you run the app without the feature flag; it should return the percentage/amounts you used to see on develop branch; if you run with the feature flag and `getIsTokenNetworkFilterEqualCurrentNetwork` returns false (when user has all networks selected) it should then show the percentage cross chains